docs/get-started/cardano-cli/governance/governance.md

```diff
@@ -105,4 +105,98 @@ cardano-cli conway transaction submit --tx-file updateNOpt.tx.signed
 
 ## Conway era update proposals
 
-TODO: Intro and diagram similar to the ones above. 
+The Conway Era introduces a new governance framework to Cardano, marking a significant evolution in how decisions about the protocol are made. Building on the foundations laid in previous eras, Conway empowers Cardano with a decentralized governance model where decision-making is accessible to stakeholders. Key features include the introduction of *Delegated Representatives (DReps)*, the *Constitutional Committee*, and an decision making role for *Stake Pool Operators* (SPOs). These entities collaborate through a formalized voting system to vote protocol paramter changes that can be proposed by any ada holder. These governance bodies are in charge of treasury funds, adding or removing members from the constitutional committee, making updates to the Cardano Constitution. Conway era makes Cardano more resilient and 100% community-driven.
+
+The process for submitting and voting update proposals undergoes significant changes in the Conway Era. First, Genesis delegations and MIR certificates are eliminated. In `DState`, the fields associated with these features are no longer included, and `DelegEnv` no longer contains the fields it had in the Shelley era, effectively rendering the genesis keys obsolete. 
+
+### Governance actions
+
+In Conway era, any ada holder can submit a Governance action, which is the on-chain method to put a proposal to the consideration of the three governance bodies. There are 7 types of governance actions:
+
+| Governance Action | Description |
+| ----------------- | ----------- |
+| NoConfidence      | A motion to create a state of no-confidence in the current constitutional committee |
+| UpdateCommittee   | Changes to the members of the constitutional committee and/or to its signature threshold and/or terms |
+| NewConstitution   | A modification to the off-chain Constitution and the proposal policy script |
```

Comment:
```suggestion
| NewConstitution   | A modification to the off-chain Constitution and the or proposal policy script |
```

Reply:
@Ryun1 can you double-check this suggestion?  Should it b:e _modification to the off-chain Constitution **or** the proposal policy script_?

Reply:
oops, I meant "and or"
I will edit

Reply:
aha, I didn't read it as the grammatically incorrect but universally accepted `and/or` ... which is logically just an `or` 🤓 (I've corrected the change above)